docs/user/alerting/stack-alerts/index-threshold.asciidoc

```diff
@@ -5,15 +5,15 @@
 The index threshold alert type is designed to run an {es} query over indices, aggregating field values from documents, comparing them to threshold values, and scheduling actions to run when the thresholds are met.
```

Comment:
```suggestion
The index threshold alert type runs an {es} query. It aggregates field values from documents, compares them to threshold values, and schedules actions to run when the thresholds are met.
```

Reply:
Index threshold
   Create the alert
   Define the conditions
   Add action variables
   Example